server/public/model/client4.go

```diff
@@ -604,6 +604,27 @@ func (c *Client4) GetServerLimits(ctx context.Context) (*ServerLimits, *Response
 	return &serverLimits, BuildResponse(r), nil
 }
 
+func (c *Client4) CreateScheduledPost(ctx context.Context, scheduledPost *ScheduledPost) (*ScheduledPost, *Response, error) {
+	buf, err := json.Marshal(scheduledPost)
+	if err != nil {
+		return nil, nil, NewAppError("CreateScheduledPost", "api.marshal_error", nil, "", http.StatusInternalServerError).Wrap(err)
+	}
+
+	r, err := c.DoAPIPost(ctx, c.postsRoute()+"/schedule", string(buf))
+	if err != nil {
+		return nil, BuildResponse(r), err
+	}
+	defer closeBody(r)
+	var createdScheduledPost ScheduledPost
+	if r.StatusCode == http.StatusNotModified {
```

Comment:
Do we expect to cache this endpoint? I'm not seeing this code explicitly returned.

Reply:
Great catch! This shouldn't be here. We do not cache the create endoint.